Tetris/main.cpp

```diff
@@ -1,4 +1,7 @@
 ﻿#include <iostream>
```

Comment:
Windows dependent stuff shall be either excluded to separate files or at least surrounded by #if...